### DISCUSSION TITLE
Suggestion: VNumberField

### DISCUSSION THREAD

**Discussion by 3dyuval** [vuetifyjs/vuetify#18451] [Ideas]
While researching how to implement a custom number component I've discovered the native input[type="number"] [handlers ](https://www.w3schools.com/jsref/met_number_stepup.asp#:~:text=The%20stepUp()%20method%20increments,use%20the%20stepDown()%20method.) which abled me to create a custom VNumberField component. 

This [demo](https://play.vuetifyjs.com/#eNrFV21v2zYQ/ius98EOZklJ231R7bR7yYANWFesazEgDlpZohw2EkWQlOOsy3/fwxdZlOwFGfqhRRHbd8e75154vLv8PFEyT74XIt62dJJOFprWoso0PV9xQhbbKBPCfrU/8obrjHEqPckSeVuvqYxKRquiIxNSZWtaLVeT15ZLfrbcSc+HkZyCf8E12E5HyE/rbAc2/g6pjBsq4wOq0lSAbD5C+jaqm8KigP6ekYTo15pHG9m0nZcPOtW79XZkihCP4TQ+PT0bcoaujkGGMMe8HukBqD2S3xhntXFvZAcx8vRev43b47Rmu+Na9/RAq8lRGNXkMKyGNqweQ3HFtUiCosNPlUsmNFFUtwKA+AY2tFpN7DFWi0Zq8plIWpJ7UsqmJlPU7jRgvndFZ2vOS8RJSDTFjgPmCEApbdwiS6Nydvbd6elJwGDcM0KqyVNHjk/PAg6CuVcE8iJxzgD6ZA4nIFSyTfxJNRyX7bM5t5rkTS1YReXvQjMoWU1SYjmGl1VVc/urpWnZ0nlHz69pfnOE/kkhFym+vJFUUbmlq8mepzO5odqxL96+pjt83zORzLaC9APMP6hqqtZgdGI/tLwA7EDOov3FJoHxzZ/qYqcpV51TBqiRvLfyqwmS8OMDrvdwn8XP7LkVx3/E0aU5qjMxiqRjDNXAioN7rbVQaZLkBccxlC7byphTnXBRJ68glsiWa1bTqGjqV7AZP08KpnRIjqmqo7VsbhFbKAlcx3nNyrtHmPKS1sLTM2fC0yJcP2VsHOhObDJlJCkvqDRBf5xLo2OhWyPWgWuDkFeM36hx3Sob6csv9dfqueosmfdofFfNw/TorjDHVaxFq2kxJ7eZzq/fNEpflCXN9TxsGC9MA7CHTWcr0eXIG9kI1ZWObW/vs6qlKcn4HcS7Nv8yxYdEhZN//MuFL7gMtER3K7xgkgCHlLBKWpVtaLon+5cCjQNtk8xuWVUBg2olJU9JQXNWZ5WyPQUgGIc1Z8QrRq8akhC3vgEJ68ISegwY69DC/j2fnViPO0FaM72Xu8APtfCOT1tRoBmnNgDR1kRgijTbL51hpAuGO50DAJlUtDDvI7R3mVjs47UP0/lsdkKW56QLNyvJTN8J2pTOh9g12eUSeLoz05NOmiDTupU8kPXh8dEIW/VbZ3z5nwacT9D+MmDFunEHgTM9tGPwPum1x4znVVtQNZvG05NDnL3ki/7lN9Vgqm+DC30M/KVlyqyao26z3PTIrLqCI4FdJSqmrdEAWHeOPDHumbANLA4KMyYsRl7JR/OUfRzBhl7ybWDcgu9LmxadXqeDfOx17H05jIAtHlR4UI0VrSk3BWnezmGlusnEcVwa+qvZSZYttwitlXcCKfMJ8IpjW70vbQIN28fKqnY8GBjJ2o/+NvNMsy0lSjCOGYbQrcFbNFQR3iCU9s64XoI4ue7hLk9wRwcwf2pu+YNAncBXgjpqnf62dmDRL2bH+sScuNY9C+CeOBfuzUcwEOHHeNnQGDvCmRsZR6P0vnaDsbnDbqQPVgZBpcLDArHIDqzXTYVXzfH6SdWp9ke6raFvWHvOwZYx3ka6qbZzgHyDDpW1lfHLD8GqQqh5VhusnonHCv6Dvh+KBwEY6BOSCrzMkc3gca0Dkf+jG2P3EdUF25K8ypTyiwhCSsqK7qIcUcWgkGP4qxGR4faAQT/cZTw0tBg5SJr79yqvWH6D2m66de3daBdiuPIm0AWLcLyN2vGulEmWcW28r7BLjFYw9rcxvvPDS79foeJERWPnSJwzmWPUDSR0tmZ4Y0yKo8HyNlqVjjuLJ/txzpr7/LC7xYHEV3N4gUlte7yYzOrWX9VjG5y+q5wo43j/L+2V7V7ZqzSNbun6humowWwgI9OmonWrdcPtyPvwGVu14ZmuJ22ZYmuGx/AuJdesKCj3DbE7aco+kxnPzRTTcEqeuLER4Q3nqHjr/PrwIbwnxHZkcye8uU0mUkwzmEe9GdEoZno7psW1XZS6hqxhU5WNrM0KhK8mTn/NorPnYuc7o+2LPmaT+/nETumYhd3IPLn6F8EXrfA=) uses VTextFIeld under the hood with a type="number" property to achieve a native like number with a custom spinner.

Is this something that could be implemented into the library?
I assume many can enjoy the benefit of having a Vuetify number input, and maybe it could be further refined by the community in other visual ways.

**Comment by scscgit** [comment]
Personal note: number input is pointless, because of its broken behavior like reacting to a mouse scroll by increasing/decreasing the value by 1, making it prone to accidents during page scrolling; plus the brower-specific arrow buttons, which may not trigger the relevant input event
![image](https://github.com/vuetifyjs/vuetify/assets/3118390/009704c0-a4b8-493b-8a22-e4a53f914d5d) < Firefox
